tutorial/datastructures.po

```diff
@@ -63,16 +63,16 @@ msgid ""
 msgstr ""
 "Insère un élément à la position indiquée. Le premier argument est la "
 "position de l'élément courant avant lequel l'insertion doit s'effectuer, "
-"donc ``a.insert(0, x)`` insère l'élément en tête de la liste, et ``a."
+"donc ``a.insert(0, x)`` insère l'élément en tête de la liste et ``a."
 "insert(len(a), x)`` est équivalent à ``a.append(x)``."
 
 #: ../Doc/tutorial/datastructures.rst:43
 msgid ""
-"Remove the first item from the list whose value is *x*.  It is an error if "
-"there is no such item."
+"Remove the first item from the list whose value is equal to *x*.  It is an "
```

Comment:
Why is the msgid changed here?

Reply:
Je ne sais pas ! j'ai "joué" avec make merge et git pull dans cpython. J'essaie de reproduire...